examples/with-tailwindcss/styles/index.css

```diff
@@ -2,8 +2,8 @@
 
 /* purgecss start ignore */
 @tailwind base;
-/* purgecss end ignore */
 @tailwind components;
+/* purgecss end ignore */
```

Comment:
Moved `/* purgecss end ignore */` 1 line down according to https://tailwindcss.com/docs/controlling-file-size#setting-up-purgecss